content/tutorials/geodesic-getting-started.md

```diff
@@ -54,47 +54,61 @@ There are a few things going on there, so let's break that down a bit:
 1. We're using the `--volume` flag to mount our `$HOME` directory to `/localhost` in our new container. This is a Geodesic standard practice which enables the container and your corresponding shell session to have access to your dotfiles, configurations, and the projects that you'll work on.
    1. **NOTE**: If you're running on Linux and using Geodesic, any files written to the `--volume` mounts will be owned by the user inside the container, which is `root`. [See here for potential workarounds](https://github.com/moby/moby/issues/3124#issuecomment-104936573).
 1. Finally, after the image name, we're passing `--login`. This is the Docker `CMD` that we're passing to our image. Since we override the Docker `ENTRYPOINT` with a small bash script, our `--login` `CMD` results in calling `/bin/bash --login` which creates a new [bash login shell](https://www.gnu.org/software/bash/manual/html_node/Bash-Startup-Files.html).
+   1. It's worth noting that since Geodesic `v0.143.2` ([PR #693](https://github.com/cloudposse/geodesic/pull/693)), you can now drop `--login` as default `CMD` will provide the same functionality.
 
 The result of running this command should look like this:
 
 ![Geodesic Login Shell](/assets/geodesic-login-shell.png)
 
-### 2. Authenticate with AWS + aws-vault
+### 2. Pull our Tutorial Project
 
-Great -- we've started up Geodesic so now we need to authenticate with AWS. To accomplish this, Geodesic ships with [`aws-vault`]({{< relref "reference/tools.md#aws-vault" >}}) to help manage our credentials and retrieve access tokens from AWS to provide us with authenticated sessions. To set up a new profile, first [create a new IAM user and programmatic Access Key ID and Secret Key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html#id_users_create_console) and be sure to copy those values down somewhere. Now, in your Geodesic shell, let's do the following:
+Great -- we've started up Geodesic so now let's do something with it. How about we pull a terraform project and apply it? To accomplish this, let's do the following:
 
 ```bash
-# Since our Geodesic on linux, let's set the default backend to an encrypted file instead of our keyring
-export AWS_VAULT_BACKEND=file
+# Change to our /localhost directory so that we can pull our project's code to our
+# local machine as well as our docker container
+cd /localhost
 
-# Add our new credentials to aws-vault. This will prompt you for the Access Key ID and Secret Key that you copied down earlier, which you should input.
-aws-vault add luke.skywalker
+# Clone our tutorials repository
+git clone https://github.com/cloudposse/tutorials
```

Comment:
I think we should keep this, but just because you asked... this is how with `terraform init`:

```
mkdir demo
terraform -chdir=demo init -from-module=git::https://github.com/Gowiem/tutorials.git//01-geodesic?ref=geodesic-tutorial
```

Note, this is the standard module notation for a `source`:
```
module "foobar" {
  source = "git::https://github.com/Gowiem/tutorials.git//01-geodesic?ref=geodesic-tutorial"
}
```

Reply:
Aha good to know -- Thanks for the info 👍